server/src/test/java/org/elasticsearch/common/rounding/TimeZoneRoundingTests.java

```diff
@@ -514,6 +514,22 @@ public void testEdgeCasesTransition() {
         }
     }
 
+    public void testDST_Europe_Rome() {
+        // time zone "Europe/Rome", rounding to days. Rome had two midnights on the day the clocks went back in 1978, and
+        // timeZone.convertLocalToUTC() gives the later of the two because Rome is east of UTC, whereas we want the earlier.
+
+        DateTimeUnit timeUnit = DateTimeUnit.DAY_OF_MONTH;
+        DateTimeZone tz = DateTimeZone.forID("Europe/Rome");
+        Rounding rounding = new TimeUnitRounding(timeUnit, tz);
+
+        long timeAfterSecondMidnight = time("1978-10-01T06:00:00+01:00");
```

Comment:
Can you add a time before the first midnight and a time between first and second just to make it easier to understand whats going on when reading the test later? I did this locally to just make sure I understand this case better, I think it helps when revisiting these tests in the future.

Reply:
Sure, no problem, I've done so.